go/parquet/file/column_writer.go

```diff
@@ -534,6 +536,13 @@ func (w *columnWriter) Close() (err error) {
 
 		w.defLevelSink.Reset(0)
 		w.repLevelSink.Reset(0)
+		if w.bitsBuffer != nil {
+			w.bitsBuffer.Release()
```

Comment:
Can there be a leak in `return err` above?

Reply:
Good catch, i should shift this above that `return err` and use a `defer` to ensure it is always called.